README.md

```diff
@@ -51,14 +51,17 @@ read data from stdin. Alternatively, you can use `cargo fmt` to format all
 binary and library targets of your crate.
 
 You'll probably want to specify the write mode. Currently, there are modes for
-replace, overwrite, display, and coverage. The replace mode is the default
-and overwrites the original files after renaming them. In overwrite mode,
-rustfmt does not backup the source files. To print the output to stdout, use the
-display mode. The write mode can be set by passing the `--write-mode` flag on
-the command line.
-
-`rustfmt filename --write-mode=display` prints the output of rustfmt to the
-screen, for example.
+diff, replace, overwrite, display, coverage, and checkstyle.
+
+* `replace` Is the default and overwrites the original files after renaming them.
+* `overwrite` rustfmt does not backup the source files.
```

Comment:
Nit: capital R. Probably better to say that `replace` does make backups, and overwrite overwrites the original file (without backup).